agent/service.go

```diff
@@ -47,23 +48,30 @@ func NewTaskService(runc shim.Shim) shim.Shim {
 func (ts *TaskService) Create(ctx context.Context, req *shimapi.CreateTaskRequest) (*shimapi.CreateTaskResponse, error) {
 	log.G(ctx).WithFields(logrus.Fields{"id": req.ID, "bundle": req.Bundle}).Info("create")
 
-	// Use mount path instead of bundle path inside the VM
-	req.Bundle = bundleMountPath
-
-	// Do not pass any mounts to runc, everything is already mounted for us
-	req.Rootfs = nil
-
 	// Passthrough runcOptions
 	opts, err := unpackBundle(filepath.Join(bundleMountPath, "config.json"), req.Options)
 	if err != nil {
 		return nil, err
 	}
 	req.Options = opts
+	// Use mount path instead of bundle path inside the VM
+	req.Bundle = bundleMountPath
+
+	// Do not pass any mounts to runc, everything is already mounted for us
+	req.Rootfs = nil
+	// TODO: handle stdio
```

Comment:
We've got a bunch of these TODO comments in our code. Can we start including links to associated GitHub issues along side them? We lose too much state otherwise.

Reply:
We could I guess. I don't think all TODO comments warrant a github issue. 

I added a link to the relevant issue for this TODO though.

Reply:
Thanks. Maybe they don't always need it, but in general, if it's something that we know needs to be done, we should document it. Issues are the right place for that.